services/ux-backend/main.go

```diff
@@ -40,6 +41,26 @@ func loadAndValidateServerConfig() (*serverConfig, error) {
 		return nil, fmt.Errorf("Malformed user-defined listening port:  %s. shutting down: %v", listenPortAsString, err)
 	}
 
+	runServerWithTLSAsString := os.Getenv("RUN_SERVER_WITH_TLS")
```

Comment:
Lets simplify the env var name 
```suggestion
	tlsEnabledAsString := os.Getenv("TLS_ENABLED")
```